java/cdk8s-core/package.json

```
@@ -7,8 +7,9 @@
   "private": true,
   "scripts": {
     "install": "mvn install",
-    "import": "cdk8s import && rm -rf imports",
-    "build": "mvn clean compile && npm run import && npm run synth",
+    "cleanImports": "rm -rf src/main/java/imports && rm -rf src/main/resources/imports && rm -rf imports",
```

Scripts in `package.json` are not usually camelCased as they are executed from shell, so should more shell like:

```suggestion
    "clean-imports": "rm -rf src/main/java/imports && rm -rf src/main/resources/imports && rm -rf imports",
```